packages/backend/src/server/web/boot.js

```diff
@@ -106,24 +106,47 @@
 	function renderError(code, details) {
 		let errorsElement = document.getElementById('errors');
 		if (!errorsElement) {
+			document.getElementsByTagName("head")[0].insertAdjacentHTML(
+				"beforeend",
+				`<link rel="stylesheet" href="../error.css" />`);
```

Comment:
I think this is the problem, loading from `../error.css` probably doesn't work because it's inserting an HTML tag.

Reply:
Also, it wouldn't even be `..` if that *was* the correct implementation, it'd be `.`. What was I thinking?